cpp/oneapi/dal/backend/transfer_dpc.cpp

```diff
@@ -101,18 +107,19 @@ sycl::event scatter_host2device(sycl::queue& q,
     auto scatter_event = q.submit([&](sycl::handler& cgh) {
         cgh.depends_on(copy_event);
 
-        byte_t* gathered_byte = reinterpret_cast<byte_t*>(gathered_device_unique.get());
-        byte_t* dst_byte = reinterpret_cast<byte_t*>(dst_device);
+        const byte_t* const gathered_byte =
+            reinterpret_cast<const byte_t*>(gathered_device_unique.get());
+        byte_t* const dst_byte = reinterpret_cast<byte_t*>(dst_device);
```

Comment:
why is 110 const byte_t* and 112 byte_t* const?

Reply:
One line is source, and other one is a consumer aka destination)